src/Bridge/Symfony/XliffFileDumper.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Translation\PlatformAdapter\PhraseApp\Bridge\Symfony;
+
+use Symfony\Component\Translation\Dumper\FileDumper;
+use Symfony\Component\Translation\MessageCatalogue;
+
+/**
+ * Custom XliffFileDumper for use with phrase app
+ */
+class XliffFileDumper extends FileDumper
```

Comment:
Unfortunately not. Symfony create a translation id with `md5($source)` - but PhraseApp uses the translation id as key (NOT the source).
Another reason is, that PhraseApp does not know `domain` so the I used this to make readable keys with domain in phraseapp: `$translation->setAttribute('id', "$domain::$source");`